Makefile

```diff
@@ -56,7 +56,8 @@ ARCHFLAGS := $(ARCHFLAGS)
 
 OPTFLAGS  ?= -g -O
 
-CFLAGS    += -Wall -Wno-unused-function -Wno-write-strings -Wno-sign-compare $(ARCHFLAGS)
+CFLAGS    += -O3 $(ARCHFLAGS)
```

Comment:
Typically a feature branch (your branch) should not change the compilation of a project, but I accept your change here.